draft-irtf-cfrg-frost.md

```diff
@@ -568,14 +549,15 @@ each Signer then runs the following procedure to produce its own signature share
   - nonce_i, pair of Scalar values (d, e) generated in round one.
   - comm_i, pair of Element values (D, E) generated in round one.
   - msg, the message to be signed (sent by the Coordinator).
-  - commitment_list = [(j, D_j, E_j), ...], a list of commitments issued by each signer,
+  - commitment_list = [(j, x, y), ...], a list of commitments issued by each signer,
     where each element in the list indicates the signer index and their
-    two commitment Element values. B MUST be sorted in ascending order
+    two commitment Element values (x, y). This list MUST be sorted in ascending order
```

Comment:
This list MUST be sorted in ascending order with respect to the participant identifier.